### PR TITLE
feat: added --merged-only option and updated usage message

### PR DESCRIPTION
**Pull request by dym-ok**

### CODE REVIEW

**Comment by pavlovic265** [comment]
For me did not work out of the box. As VS code does not clean branches that have an origin, it keeps them in an index or something like that, so I had to run ` git fetch --prune` first, and then `--merge-only` would work.

So did you have the same issue with web-storm or is it VS code thing?

**Comment by dym-ok** [comment]
You're right, I did git fetch -p manually. Added it to the command.

File: remove-git-branch [hunk 1]
```diff
@@ -35,13 +35,18 @@ for i in "$@"; do
...
+      ;;
     -h|--help)
     echo "usage: remove-git-branch  [-b=<pattern> | --branch=<pattern>] [-p=<pattern> | --pattern=<pattern>]
-                [-a | --all] [-h | --help]"
+                [-e | --exclude] [-a | --all] [-m | --merged-only] [-c | --check] [-f | --force] [-h | --help]"
       echo "-b (--branch) - Branches to remove separated by ','. It has priority over pattern."
       echo "-p (--pattern) - Regex pattern to define witch branches should be removed."
-      echo "-a (--all) - Remove all branches, expect master, **main, and current."
       echo "-e (--exclude) - Branches to exclude from deleting."
+      echo "-a (--all) - Remove all branches, expect master, **main, and current."
```
Comment (pavlovic265) [inline remove-git-branch]: Could you also remove `**` from `main`? 🙏 
Not sure why is there, I think it is some leftover.

File: remove-git-branch [hunk 2]
```diff
@@ -35,13 +35,18 @@ for i in "$@"; do
...
+      echo "-m (--merged-only) - Branches that are already merged and deleted at origin."
```
Comment (pavlovic265) [inline remove-git-branch:49]: What do you think about adding info that with this flag `git fetch --purne` will run? 🤔

File: remove-git-branch (hunk 1 above)
Comment (dym-ok) [reply remove-git-branch]: Sure.

File: remove-git-branch (hunk 2 above)
Comment (dym-ok) [reply remove-git-branch:49]: Done, added it to the usage info.

**Review by pavlovic265 — APPROVED** [review]
🙌

**Pull request merged**